src/mmw/js/src/core/models.js

```diff
@@ -223,7 +225,11 @@ var LayersCollection = Backbone.Collection.extend({
             _.each(settings.get(options.type), function(layer) {
                 var layerModel = new LayerModel();
                 layerModel.buildLayer(layer, options.type, options.initialActive);
-                self.add(layerModel);
+                if (!settings.get('data_catalog_enabled')) {
```

Comment:
These two `if` conditions can be combined:

```js
if (!settings.get('data_catalog_enabled') || layerModel.get('bigCZ')) {
  self.add(layerModel);
}
```

Reply:
Yeah -- I considered doing this and thought it'd be slightly less straightforward to read, but I may join them now that we've commented on it here.

Reply:
I inlined the checks as presented above, then tested everything out again on my local. Still works as expected, so I pushed an updated commit.